trySwift/SpeakerDetailViewController.swift

```diff
@@ -11,7 +11,7 @@ import TrySwiftData
 
 class SpeakerDetailViewController: UITableViewController {
 
-    var speaker: Speaker!
```

Comment:
Hmmm. I agree with your reasoning, but when we want to show this detail VC with splitview, the app crashes with this being force unwrapped. So we can't really "force" there being a value.